### ISSUE TITLE
Opaque error message if npm is not installed

### ISSUE THREAD

**Issue by ellisonbg** [jupyterlab/jupyterlab#3565]
In my deployment of jupyterlab+jupyterhub, I got this error message when I started to install extensions:

```
> '' pack @jupyterlab/geojson-extension
expected str, bytes or os.PathLike object, not NoneType
```

This was a combination of two things:

1. `conda install jupyterlab` didn't install npm. Is this our intended behavior?
2. The error handling for a missing npm when a user runs a lab command requiring it is a bit opaque.

Even if we don't require npm in the conda package, we should clean up that error message with something like:

"Building JupyterLab or installing JupyterLab extensions requires installing npm. If you are using conda, that can be done with `conda install nodejs`"

**Comment by blink1073** [comment]
Which version of JuptyerLab was this?  No, we don't want to override a local npm if the user has one installed.

**Comment by blink1073** [comment]
We should be catching an error when there is no npm, cf https://github.com/jupyterlab/jupyterlab/pull/3317.

**Comment by ellisonbg** [comment]
Ahh, this is on my teaching deployment, which is am using 0.30.6 for for now. Feel free to close if this is fixed in master.

**Comment by blink1073** [comment]
Hmm, that should have been fixed in one of the 0.30.x releases.  I'll double-check the code path.
